specs.wdio/quota_upgrade/paddle.spec.ts

```
@@ -0,0 +1,21 @@
+import { expect } from 'chai';
+import Session from '../helpers/session';
+
+describe('paddle notification events', () => {
+  beforeEach(Session.truncateDB);
+  afterEach(Session.afterEach);
+  after(Session.deleteBrowsers);
+
+  it.only('is possible to retrieve the quota information of a user', async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)